src/test/java/athena/commands/DeleteCommandTest.java

```diff
@@ -49,43 +48,41 @@ public void execute_invalidIndex_returnsInvalidIndexMessage() {
     /**
      * Creates a new delete command.
      *
-     * @param targetVisibleIndex of the person that we want to delete
+     * @param targetIndex of the task that we want to delete
      */
-    private DeleteCommand createDeleteCommand(int targetVisibleIndex) {
-        DeleteCommand command = new DeleteCommand(targetVisibleIndex);
+    private DeleteCommand createDeleteCommand(int targetIndex) {
+        DeleteCommand command = new DeleteCommand(targetIndex);
         return command;
     }
 
     /**
-     * Executes the command, and checks that the execution was what we had expected.
+     * Executes the command, and checks that the execution was what we expect.
      */
-    private void assertCommandBehaviour(DeleteCommand deleteCommand, String expectedMessage,
-                                        TaskList expectedTaskList, TaskList actualTaskList) {
-        Ui ui;
-        Command result = DeleteCommand.execute(taskList, ui);
-        assertEquals(expectedMessage, result);
+    private void assertCommandBehaviour(DeleteCommand deleteCommand, String expectedMessage, TaskList expectedTaskList, TaskList actualTaskList) {
+        Ui ui = null;
```

Comment:
`deleteCommand.execute` will call `ui.printTaskDeleted`, but `ui` is `null` so it will crash.